README.md

```diff
@@ -0,0 +1,60 @@
+# Gitea Operator
+
+[![Build Status](https://travis-ci.org/integr8ly/gitea-operator.svg?branch=master)](https://travis-ci.org/integr8ly/gitea-operator)
+
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0                      |
+| IRC             | [#integreatly](https://webchat.freenode.net/?channels=integreatly) channel in the [freenode](http://freenode.net/) network. |
+
+An Operator that installs Gitea and, optionally on OpenShift, an oauth proxy. Installation is performed by creating a custom resource of kind `Gitea`. You can uninstall Gitea by removing this resource.
+The Operator will also watch all Gitea resources and reinstall them if they are deleted.
+
+## Installing the Operator
+
+First we need to create a Service Account, Role and Role Binding in order to grant the required permissions to the Operator. Make sure you are logged in with a user that has permission to create those resources.
+
+```sh
+$ oc login -u system:admin
+$ oc create -f deploy/service_account.yaml
```

Comment:
We can probably put the `service_account`, `role` and `role_binding` into one rbac file.

Reply:
I'll look into that. Previous versions of the operator sdk used to do that but 0.0.7 creates service account, role and role binding as separate resources for some reason.

Reply:
I've added Makefile targets to automate that instead: https://github.com/integr8ly/gitea-operator/pull/1/commits/21ba5667ab4a4c099b81d0dc131dc274c2a49efa